PatientRegistrationApp/package.json

```diff
@@ -13,6 +13,7 @@
     "qrcode": "^1.4.4",
     "v-mask": "^2.2.3",
     "vue": "^2.6.11",
+    "vue-qrcode": "^0.3.5",
```

Comment:
Seeing some console errors here 
![image](https://user-images.githubusercontent.com/67079030/103427584-af617500-4b87-11eb-9f5e-462de39373ee.png)

Reply:
These as well 
![image](https://user-images.githubusercontent.com/67079030/103427613-d61fab80-4b87-11eb-814e-41611fe80545.png)

Reply:
I believe this is maybe becuse I used a different library without removing the first one. I can try and delete the "qrcode" library that i'm not using anymore and will check the console